src/Appwrite/Extend/Exception.php

```diff
@@ -177,6 +177,7 @@ class Exception extends \Exception
     public const DOMAIN_NOT_FOUND                  = 'domain_not_found';
     public const DOMAIN_ALREADY_EXISTS             = 'domain_already_exists';
     public const DOMAIN_VERIFICATION_FAILED        = 'domain_verification_failed';
+    public const DOMAIN_VARIABLE_INCORRECT         = 'domain_variable_incorrect';
```

Comment:
I'd use `DOMAIN_TARGET_INVALID` and `domain_target_invalid` as the key and value for this error